ui/packages/app/src/components/Button/Button.svelte

```diff
@@ -8,8 +8,16 @@
 	export let visible: boolean = true;
 	export let value: string;
 	export let variant: "primary" | "secondary" = "primary";
+	export let mode: "static" | "dynamic";
```

Comment:
If we add this, the TypeScript error should go away:

```suggestion
	export let mode: "static" | "dynamic" = "dynamic";
```